test/lib/rule-finder.js

```diff
@@ -19,6 +19,14 @@ var getRuleFinder = proxyquire('../../src/lib/rule-finder', {
     '@noCallThru': true,
     '@global': true,
   },
+  '@someone/eslint-plugin-something': {
+    rules: {
+      'one-rule': true,
+      'two-rule': true,
```

Comment:
I would stick to `foo`, `bar` rules for consistency.
**And even modified the identifier names from `react` to `plugin` and `something` to `scoped-plugin`.**

When we revisit the code later on, the names will be self explanatory.